railties/test/command/application_test.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+require "rails/command"
+
+class Rails::Command::ApplicationTest < ActiveSupport::TestCase
+  test "rails new without path prints help" do
+    output = capture(:stdout) do
+      Rails::Command.invoke(:application, %w[new])
```

Comment:
@eileencodes the app_generator_test tests the thor command directly, so I needed to put my test a few layers higher in the stack.